src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASingleFileApp.cs

```diff
@@ -528,15 +528,19 @@ public void It_runs_single_file_apps(string targetFramework, bool selfContained,
             var testAsset = _testAssetsManager.CreateTestProject(testProject);
```

Comment:
```suggestion
            var testAsset = _testAssetsManager.CreateTestProject(testProject, identifier: targetFramework + "_" + selfContained + "_" + bundleOption);
```

The retry is a good idea, but I think that adding the appropriate identifier here will likely stop this test from being flaky so we won't need it.

Reply:
https://github.com/dotnet/sdk/pull/15571